src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -213,7 +213,7 @@ public bool run_action(ChocolateyConfiguration configuration, PackageResult pack
                     this.Log().Info(ChocolateyLoggers.Important, () => @"choco feature enable -n allowGlobalConfirmation");
 
                     var selection = InteractivePrompt.prompt_for_confirmation(@"Do you want to run the script?",
-                        new[] { "yes", "no", "print" },
+                        new[] { "yes", "all - yes to all", "no", "print" },

```

Comment:
Would `always` be a better option here?

Reply:
This is in the context of that run only. Always implies that it would always do so on future runs.

Reply:
I wanted Yes to All, but I wanted the short answer `a` more so. The Interactive Prompt automatically uses the first letter to make the short answer and two "Y"s would cause it to be confused or produce unexpected results. This is effectively what it would produce: 
~~~
[Y]es
[A]ll - yes to all
[N]o
[P]rint
~~~

Reply:
To answer with the full bit, you would need to type back in either `a` or `all - yes to all`. The latter would be a bit weird and unexpected. Perhaps we should cut it off at `-` for matching.